### PR TITLE
Update/fetch key from server

### PR DESCRIPTION
**Pull request by JWZeta**

### CODE REVIEW

File: src/util/key_table.c [hunk 1]
```diff
@@ -86,10 +86,12 @@ TableResponseStatus key_table_add_from_handle(charbuf key_id, uint64_t handle)
   return status;
 }
 
-TableResponseStatus key_table_add_from_server(charbuf key_id, charbuf server_id, charbuf server_key_id)
+TableResponseStatus key_table_add_from_server(charbuf key_id, charbuf server_id, int port, charbuf server_key_id)
 {
   TableResponseStatus status;
   charbuf key;
+  const char *server_name = "localhost";
```
Comment (b-carr) [inline src/util/key_table.c:93]: After we merge this one in we'll open an issue to un-hardcode these.

File: Makefile [hunk 2]
```diff
@@ -523,6 +536,9 @@ test: all
 	@./bin/pelz seal test/data/server_cert_test.der -o test/data/server_cert_test.der.nkl
 	@./bin/pelz seal test/data/client_priv_test.der -o test/data/client_priv_test.der.nkl
 	@echo "GEN => Test Key/Cert Files"
+	@cd kmyth/sgx/demo && make pre bin/ecdh-server bin/ecdh-client
```
Comment (b-carr) [inline Makefile:539]: There's no need to build the client here.

File: Makefile [hunk 3]
```diff
@@ -523,6 +536,9 @@ test: all
...
+	@./kmyth/sgx/demo/bin/ecdh-server -r test/data/server_priv_test.pem -u test/data/client_cert_test.pem -p 7000 &
```
Comment (b-carr) [inline Makefile:540]: I'd prefer to redirect the server output here to ```/dev/null```, I don't think seeing its logging adds anything useful.

File: test/src/util/test_pelz_uri_helpers.c [hunk 4]
```diff
@@ -42,27 +43,37 @@ void test_scheme_extraction(void)
 
   uriParseSingleUriA(&uri, pelz_uri, NULL);
   CU_ASSERT(get_uri_scheme(uri) == PELZ_URI);
+  pelz_log(LOG_DEBUG, "Finish URI functions");
```
Comment (b-carr) [inline test/src/util/test_pelz_uri_helpers.c:46]: Why are we logging in the tests rather than relying on the test outputs?

**Review by b-carr — COMMENTED** [review]
I think we also need to update INSTALL.md to reflect the dependency on libkmip.

File: test/src/util/test_pelz_uri_helpers.c (hunk 4 above)
Comment (JWZeta) [reply test/src/util/test_pelz_uri_helpers.c:46]: This is logged so if a change causes the test to fault then you can better determine the fault location from the debug logs.

**Review by b-carr — APPROVED** [review]

**Pull request merged**